build.sh

```diff
@@ -1,23 +1,23 @@
 #!/bin/bash
-DIR=$(dirname $0)
-PROJECT="$(pwd)/${DIR}"
+set -e
 
 rustup target add aarch64-unknown-linux-gnu
 
-sudo apt-get update > /dev/null
-sudo apt install -y gcc-aarch64-linux-gnu
+apt-get update > /dev/null
+apt install -y gcc-aarch64-linux-gnu
 
-CONFIG_FILE="$HOME/.cargo/config"
+CONFIG_FILE="/root/.cargo/config"
 
 if ! grep -q "\[target.aarch64-unknown-linux-gnu\]" "$CONFIG_FILE" 2>/dev/null || ! grep -q 'linker = "aarch64-linux-gnu-gcc"' "$CONFIG_FILE" 2>/dev/null; then
     mkdir -p "$(dirname "$CONFIG_FILE")"
     {
         echo '[target.aarch64-unknown-linux-gnu]'
         echo 'linker = "aarch64-linux-gnu-gcc"'
     } >> "$CONFIG_FILE"
-    ecjo
+    echo
 fi
 export TARGET_CC=aarch64-linux-gnu-gcc
 
+cd "/systemd-crash-reporter"
```

Comment:
If you don't do this but use the pwd of this script we can still build it locally without a docker file around it and with reduced privileges

Reply:
Yes. We are doing it in multiple locations already